mempool/reactor.go

```diff
@@ -97,12 +87,6 @@ func (memR *Reactor) AddPeer(peer p2p.Peer) {
 	}
 }
 
-// RemovePeer implements Reactor.
-func (memR *Reactor) RemovePeer(peer p2p.Peer, _ interface{}) {
-	memR.ids.Reclaim(peer)
-	// broadcast routine checks if peer is gone and returns
-}
```

Comment:
Same here, this function is publicly exported, so its removal needs to be documented.